VERSION

```diff
@@ -1 +1 @@
-1.3.2
\ No newline at end of file
+1.3.2-etleap
```

Comment:
Convention is to make the branch name 1.3.2-etleap and the version 1.3.2.0-etleap - that way we can continue to evolve without interfering with the upstream version number.